windows_386.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017 The Virtual Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+//go:generate go run generator.go
```

Comment:
Please verify that `go generate -v` runs the gereator only once. If not, then this line should be removed.